doc/binder/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# For binder
```

Comment:
Should we link to one or multiple files from https://github.com/scikit-image/scikit-image/tree/master/requirements to reduce the maintenance demand here? Similarly to https://github.com/scikit-image/scikit-image/blob/master/requirements.txt

Reply:
It's a good idea I'll move the file to the `requirements` directory (I think we need a separate file though because we don't need all the doc-building dependencies). Thanks!